src/libs/ReportUtils.js

```diff
@@ -1656,12 +1684,14 @@ function getReportPreviewMessage(report, reportAction = {}, shouldConsiderReceip
         }
     }
 
-    if (isSettled(report.reportID)) {
+    // Show Paid preview message if it's settled or if the amount is paid & stuck at receivers end for only chat reports.
+    if (isSettled(report.reportID) || (report.isWaitingOnBankAccount && isPreviewMessageForParentChatReport)) {
         // A settled report preview message can come in three formats "paid ... elsewhere" or "paid ... with Expensify"
         let translatePhraseKey = 'iou.paidElsewhereWithAmount';
         if (
             _.contains([CONST.IOU.PAYMENT_TYPE.VBBA, CONST.IOU.PAYMENT_TYPE.EXPENSIFY], lodashGet(reportAction, 'originalMessage.paymentType')) ||
-            reportActionMessage.match(/ (with Expensify|using Expensify)$/)
+            reportActionMessage.match(/ (with Expensify|using Expensify)$/) ||
+            report.isWaitingOnBankAccount
```

Comment:
Can you make sure that this OR condition doesn't cause regression of showing `paid with Expensify` for even non paid action?

Reply:
Non-paid action in-the sense?

Reply:
Updated comment.

Reply:
Yes, I think `isSettled` would do the work.

Reply:
ok, so ideally this should be fixed in backend. Given urgency, frontend hot-fix would be fine here.